### ISSUE TITLE
Handling null or undefined input in `lowercaseKeys` function

### ISSUE THREAD

**Issue by palashmon** [sindresorhus/lowercase-keys#14]
I noticed another potential bug in the `lowercaseKeys` function regarding how it handles `null` or `undefined` input. Currently, if the function receives null or undefined as input, it throws an error due to attempting to access the properties of an undefined or null object.

To reproduce the issue:

```js
console.log(lowercaseKeys(null)); 
// Uncaught TypeError: Cannot convert undefined or null to object
```

Thank you!

**Comment by sindresorhus** [comment]
What did you expect to happen?

**Comment by palashmon** [comment]
Maybe we could consider returning an empty object instead of displaying the error.

**Comment by sindresorhus** [comment]
I think it's better to throw a TypeError with a human-friendly error message. Silently returning an empty object could cause problems later on.

**Comment by palashmon** [comment]
Yes, I agree. Adding a TypeError with a clear error message would provide better feedback to users.
